inspector_go/rpc/tx_decode.go

```diff
@@ -0,0 +1,44 @@
+package rpc
+
+import (
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	txtypes "github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/umee-network/price_inspector/config"
+	"github.com/umee-network/price_inspector/utils"
+	"github.com/umee-network/umee/v6/app"
+	otypes "github.com/umee-network/umee/v6/x/oracle/types"
+)
+
+// TxDecoder decodes a base64 encoded transaction data and returns a slice of ValidatorExgRates and an error if the decoding fails.
+func TxDecoder(txData string, codec testutil.TestEncodingConfig) ([]config.ValidatorExgRates, error) {
+	d, err := utils.DecodeBase64(txData)
+	if err != nil {
+		return nil, err
+	}
+	codec = app.MakeEncodingConfig()
+	var c txtypes.Tx
+	err = codec.Codec.Unmarshal(d, &c)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
is there a ready function in SDK to unpack transaction?

Reply:
I don't think so